main.js

```diff
@@ -0,0 +1,77 @@
+let books = [];
+
+const book = document.getElementById('book');
+
+function Book(title, author) {
+  this.title = title;
+  this.author = author;
+}
+
+function addBook() {
+  for (let i = 0; i < books.length; i += 1) {
+    const bookData = `
+        <div id="book${i}">
+            <h2>${books[i].title}</h2>
+            <p>${books[i].author}</p>
+            <button id="${i}" class="remove">x</button>
+            <hr>
+        </div>
+        `;
+    book.innerHTML += bookData;
+  }
+}
+
+addBook();
+
+const forma = document.querySelector('form');
+const suma = 1;
+
+forma.addEventListener('submit', (e) => {
+  e.preventDefault();
+  let id = 0;
+
+  if (books.length === 0) {
+    id = 1;
+  } else {
+    id = [(books.length - 1) + suma];
+  }
+  const t = document.getElementById('title').value;
+  const a = document.getElementById('author').value;
+  const y = new Book(t, a);
+  books.push(y);
+  const bookData = `
+    <div id="${id}">
+        <h2>${y.title}</h2>
+        <p>${y.author}</p>
+        <button id="${y}" class="remove">x</button>
+        <hr>
+    </div>
+    `;
+  book.innerHTML += bookData;
+  const titleValue = document.getElementById('title');
+  const authorValue = document.getElementById('author');
+  titleValue.value = '';
+  authorValue.value = '';
+  localStorage.setItem('books', JSON.stringify(books));
+});
```

Comment:
- [ ] You have a function for displaying books on your html page 😊 But it would be nice if you would do the same thing with adding a book function 👍 As you can see below ⬇️

```javascript
const addSingleBook = () => {
  let id = 0;

  if (books.length === 0) {
    id = 1;
  } else {
    id = [(books.length - 1) + suma];
  }
  const t = document.getElementById('title').value;
  const a = document.getElementById('author').value;
  const y = new Book(t, a);
  books.push(y);
  const bookData = `
    <div id="${id}">
        <h2>${y.title}</h2>
        <p>${y.author}</p>
        <button id="${y}" class="remove">x</button>
        <hr>
    </div>
    `;
  book.innerHTML += bookData;
  const titleValue = document.getElementById('title');
  const authorValue = document.getElementById('author');
  titleValue.value = '';
  authorValue.value = '';
  localStorage.setItem('books', JSON.stringify(books));
};

forma.addEventListener('submit', (e) => {
  e.preventDefault();
  addSingleBook();
});
```